spk/emby/Makefile

```diff
@@ -0,0 +1,52 @@
+SPK_NAME = emby
+SPK_VERS = 3.0.5597.1
+SPK_REV = 3
+SPK_ICON = src/emby.png
+DSM_UI_DIR = app
+
+BETA = 1
+
+DEPENDS = cross/emby cross/busybox cross/sqlite
+
+SPK_DEPENDS = "mono>3.6:imagemagick=6.9.0-1:mediainfo=0.7.72-1:ffmpeg=2.6-3"
+
+MAINTAINER = SynoCommunity
+
+DESCRIPTION = Emby streams everything Your personal videos, music, photos, and live television in the palm of your hand.
+ADMIN_PORT = 8096
+RELOAD_UI = yes
+DISPLAY_NAME = Emby
+CHANGELOG = "1. Update emby to 3.0.5597.1<br>2. depend on ffmpeg >= 2.6-3"
```

Comment:
Yes, but imo that means you need to update both packages at the same time when major upgrades to ImageMagick occurs, which is understandable. 
I was thinking more along the lines of updating ImageMagick to `imagemagick 6.9.0-2`, or a minor update to say 6.9.1 which doesn't necessarily mean the patch needs changing.